deblur/workflow.py

```diff
@@ -82,27 +91,31 @@ def build_index_sortmerna(ref_fp, working_dir):
     ref_fp: tuple
         filepaths to FASTA reference databases
     working_dir: string
-        working directory path
+        working directory path where to store the indexed database
 
     Returns
     -------
     all_db: tuple
         filepaths to SortMeRNA indexed reference databases
-    all_file_to_remove: list
-        index files to remove
     """
+    logger = logging.getLogger(__name__)
+    logger.info('build_index_sortmerna files %s to'
+                ' dir %s' % (ref_fp, working_dir))
+
     all_db = []
-    all_files_to_remove = []
     for db in ref_fp:
-        # build index
-        sortmerna_db, files_to_remove = \
-            build_database_sortmerna(
-                fasta_path=db,
-                max_pos=10000,
-                output_dir=working_dir)
-        all_db.append(sortmerna_db)
-        all_files_to_remove.extend(files_to_remove)
-    return tuple(all_db), all_files_to_remove
+        fasta_dir, fasta_filename = split(db)
+        index_basename = splitext(fasta_filename)[0]
+        db_output = join(working_dir, index_basename)
+        logger.debug('processing file %s into location %s' % (db, db_output))
+        params = ['indexdb_rna', '--ref', '%s,%s' % (db, db_output), '--tmpdir', working_dir]
+        res = subprocess.call(params)
```

Comment:
Same comment as above (applies to all - change all `subprocess.call` to `system_call`.